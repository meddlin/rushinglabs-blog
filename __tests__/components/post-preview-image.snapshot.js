import { render } from '@testing-library/react';
import PostPreviewImage from '../../components/post-preview-image';

it('renders PostPreviewImage unchanged', () => {
    const { container } = render(<PostPreviewImage />);

    expect(container).toMatchSnapshot()
})