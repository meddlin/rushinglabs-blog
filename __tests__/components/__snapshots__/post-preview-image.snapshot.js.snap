// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders PostPreviewImage unchanged 1`] = `
<div>
  <div
    class="flex flex-col"
  >
    <div />
    
  </div>
</div>
`;
